backend/testfiles/execution/language/elambda.dark

```diff
@@ -2,43 +2,52 @@ PACKAGE.Darklang.Stdlib.List.push_v0 [] (fun x -> -4.611686018e+18) = [ (fun x -
                                                                           -4.611686018e+18) ]
 
 // Test that empty parameters are removed
-(let y = (fun x ___ -> x + 1) in Builtin.List.map_v0 [ 1; 2; 3; 4 ] y) = [ 2
-                                                                           3
-                                                                           4
-                                                                           5 ]
+(let y = (fun x ___ -> x + 1) in PACKAGE.Darklang.Stdlib.List.map_v0 [ 1; 2; 3; 4 ] y) = [ 2
+                                                                                           3
+                                                                                           4
+                                                                                           5 ]
 
-(let y = (fun x -> x + 1) in Builtin.List.map_v0 [ 1; 2; 3; 4 ] y) = [ 2; 3; 4; 5 ]
+(let y = (fun x -> x + 1) in PACKAGE.Darklang.Stdlib.List.map_v0 [ 1; 2; 3; 4 ] y) = [ 2
+                                                                                       3
+                                                                                       4
+                                                                                       5 ]
 
 (let y = (fun msg -> Builtin.Test.runtimeError msg) in
- Builtin.List.map_v0 [ "1"; "2"; "3"; "4" ] y) = Builtin.Test.derrorMessage "1"
+ PACKAGE.Darklang.Stdlib.List.map_v0 [ "1"; "2"; "3"; "4" ] y) = Builtin.Test.derrorMessage
```

Comment:
re-check this test

Reply:
Yeah, your implementation does the last items first, but i think the previous implementation did the first items first.

I think it would make more sense for it to do the first elements of the list first, but there may be reasons that isn't feasible?

Reply:
This was only happening with that error case; I didn't have to change the results of the other tests. Updating List.map to use List.fold solved this issue